docs/tutorials/go-built-in.md

```diff
@@ -48,7 +48,8 @@ have installed and the computer platform):
 
 ```bash
 $ go version
-go version go1.21.8 darwin/amd64
+go version go1.22.2 darwin/amd64
+
 ```
 
 ## 1.1 Installing CometBFT
```

Comment:
should update here too https://github.com/cometbft/cometbft/blob/a46c5aa064826c648408b4e93b64e535c31233fc/docs/tutorials/go-built-in.md?plain=1#L144

Reply:
moment